BACKEND/docker-compose.yml

```diff
@@ -4,7 +4,7 @@ services:
     image: postgres
     restart: always
     ports:
-      - 5433:5432
+      - 5432:5432
```

Comment:
File of a century, we dont know how many time this fucking file has been modified 🥲